lxd/instances_put.go

```diff
@@ -90,26 +102,120 @@ func instancesPut(d *Daemon, r *http.Request) response.Response {
 
 	// Batch the changes.
 	do := func(op *operations.Operation) error {
+		localAction := func(local bool) error {
```

Comment:
@stgraber just a nit, but it feels like `local` arg could be better named here, as its meaning isn't clear, after all the function itself is called `localAction`, so what would `localAction` mean when `local` arg is false?

Reply:
Perhaps this argument should be called `clusterNotification` instead. This seems to alter the behaviour depending on whether it was called on the origin node or the notified node.